installer/bundle/bundle_skel.sh

```diff
@@ -61,6 +61,8 @@ UNSUPPORTED_OPENSSL=60
 INSTALL_PYTHON_CTYPES=61
 INSTALL_TAR=62
 INSTALL_SED=63
+INSTALL_CURL=64
+INSTALL_GPG=65
```

Comment:
Added new bundle error codes in omsagent repo in this PR. I am going to leave it to you to remove duplicated codes from extension repo since you originally created the table. Thanks.